go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/cybozu-go/well v1.10.0
 	github.com/go-logr/logr v0.4.0
 	github.com/google/go-cmp v0.5.6
-	github.com/kubernetes-csi/csi-test/v4 v4.2.0
+	github.com/kubernetes-csi/csi-test/v4 v4.2.1-0.20210804222850-c5c0c94cd86e
```

Comment:
csi-test is not support RWOP yet completely.
https://kubernetes.slack.com/archives/C09QZFCE5/p1628699178102700?thread_ts=1628698823.102500&cid=C09QZFCE5